src/views/admin/UserDetailsView.vue

```diff
@@ -25,14 +27,17 @@
         password.value = newPassword || ''
       })
       .catch((error: string) => {
-        errorCode.value = error
+        personStore.errorCode = error
```

Comment:
Der Fehler wird im Store schon behandelt und in diese Variable geschrieben.

Reply:
Weg gemacht